src/services/api/resources/chats/__tests__/linkContact.unit.spec.js

```diff
@@ -0,0 +1,101 @@
+import { vi, describe, it, expect } from 'vitest';
+import linkContact from '../linkContact';
+import http from '@/services/api/http';
+import { getProject } from '@/utils/config';
+
+vi.mock('@/services/api/http');
+vi.mock('@/utils/config', () => ({
+  getProject: vi.fn(() => 'mocked-project-id'),
+}));
+
+describe('Link Contact Service', () => {
+  const uuid = getProject();
+
+  describe('getLinketContact', () => {
+    it('should make a GET request to /project/{uuid}/retrieve_linked_contact/ with correct params', async () => {
+      const mockResponse = { data: { linkedContact: true } };
+      http.get.mockResolvedValue(mockResponse);
+
+      const contact = { id: 'contact-id' };
```

Comment:
you could avoid reproducing `const contact...` in all tests if you put it in the scope of the description of the `Link Contact Service`